### ISSUE TITLE
Hormone receptors positive in breast

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10087]
**Preferred term label:**
Hormone receptors positive in breast (it may make sense to split into separate estrogen and progesterone findings as in the UMLS entries below)

**Synonyms**
Estrogen and progesterone

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C0279759
https://uts.nlm.nih.gov/uts/umls/concept/C0279754

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Breast ductal carcinoma in situ

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding
Increased breast-tissue estrogen receptor expression
HP:0020325
Def: Increased level of estrogen receptor expression in breast tissue.
Comment: More than 70% of all breast cancers are classified as ER + based on the detection of ER expression by immunohistochemistry in at least 1% of the tumor cells.
PMID: 35668111

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding
Increased breast-tissue estrogen receptor expression
HP:0020325
Def: Increased level of estrogen receptor expression in breast tissue.
Comment: More than 70% of all breast cancers are classified as ER + based on the detection of ER expression by immunohistochemistry in at least 1% of the tumor cells.
PMID: 35668111

**Comment by pnrobinson** [comment]
@MickeySegal 
adding
Increased breast-tissue progesterone receptor expression
HP:0020326
def:Increased level of progesterone receptor expression in breast tissue.
Comment: Almost 70% of breast cancers are hormone receptor-positive. Their cells have positive expression of estrogen receptor (ER) and/or progesterone receptor (PR), which are related to cancer cell growth and spread. Estrogen and its receptor, ER, play a critical role in the development and progression of breast cancer. PR is an upregulated target gene of ER, its expression is dependent on estrogen, and PR can modulate ER action. PR is also a valuable prognostic biomarker of overall survival or disease-free survival in breast cancer.
PMID:35115765
